mne/decoding/ems.py

```diff
@@ -1,26 +1,111 @@
 # Author: Denis Engemann <denis.engemann@gmail.com>
 #         Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#         Jean-Remi King <jeanremi.king@gmail.com>
 #
 # License: BSD (3-clause)
 
 import numpy as np
 
+from .mixin import TransformerMixin, EstimatorMixin
+from .base import _set_cv
 from ..utils import logger, verbose
 from ..fixes import Counter
 from ..parallel import parallel_func
 from .. import pick_types, pick_info
 
 
+class EMS(TransformerMixin, EstimatorMixin):
+    """Transformer to compute event-matched spatial filters.
+
+    This version operates on the entire time course. The result is a spatial
+    filter at each time point and a corresponding time course. Intuitively,
+    the result gives the similarity between the filter at each time point and
+    the data vector (sensors) at that time point.
+
+    .. note : EMS only works for binary classification.
+
+    References
+    ----------
+    [1] Aaron Schurger, Sebastien Marti, and Stanislas Dehaene, "Reducing
+        multi-sensor data to a single time course that reveals experimental
+        effects", BMC Neuroscience 2013, 14:122
+
+    Attributes
+    ----------
+    filters_ : ndarray, shape (n_channels, n_times)
+        The set of spatial filters.
+    classes_ : ndarray, shape (n_classes,)
+        The target classes.
+    """
+
+    def __repr__(self):
+        if hasattr(self, 'filters_'):
+            return 'EMS: fitted with %i filters on %i classes.' % (
```

Comment:
Please encapsulate the repr with <>

so just:

return '<EMS: fitted with %i filters on %i classes>' % ...